libpivot/include/pivot/builtins/systems/PhysicSystem.hxx

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <pivot/ecs/Core/Systems/description.hxx>
+
+namespace pivot::builtins::systems
+{
+extern const pivot::ecs::systems::Description physicSystem;
```

Comment:
Same as above

Reply:
Do you think putting all the implementation in the header would be better ?